flow_screen_components/richTextInputFSC/force-app/main/default/lwc/inputRichTextFSC/inputRichTextFSC.js

```diff
@@ -1,9 +1,215 @@
+            if(this.autoReplaceMap != undefined){
+                this.replaceMap = JSON.parse(this.autoReplaceMap);
+                this.autoReplaceEnabled = true;
+            } 
+            this.isValidCheck = true;
+            //if warn only is set, then change toast type/mode
+            if(this.warnOnly){
+                this.disallowedType = 'warning';
+                this.disallowedMode = 'dismissable';
+            }
+        }
+    }
+
+    //Handle updates to Rich Text field
+    handleTextChange(event) {
+        if (this.symbolsNotAllowed != undefined || this.wordsNotAllowed != undefined) {
+            this.interimValue = (event.target.value).toLowerCase();
+            this.interimValue = this.interimValue.replace(/(<([^>]+)>)/ig, "");
+
+            if(this.symbolsNotAllowed != undefined){
+                if (this.symbolsNotAllowed.test(this.interimValue)) {
+                    this.isValidCheck = false;
+                    const evt = new ShowToastEvent({
+                        title: this.symbolTitle,
+                        message: this.disallowedSymbolsMessage,
+                        variant: this.disallowedType,
+                        mode: this.disallowedMode
+                    });
+                    this.dispatchEvent(evt);
+                }else{
+                    this.isValidCheck = true;
+                    this.richText = event.target.value;
+                }
+            }
+
+            if(this.wordsNotAllowed != undefined){
+                if (this.wordsNotAllowed.test(this.interimValue)) {
+                    this.isValidCheck = false;
+                    const evt = new ShowToastEvent({
+                        title: this.wordTitle,
+                        message: this.disallowedWordsMessage,
+                        variant: this.disallowedType,
+                        mode: this.disallowedMode
+                    });
+                    this.dispatchEvent(evt);
+                }else{
+                    this.isValidCheck = true;
+                    this.richText = event.target.value;
+                }
+            }
+        } else {
+            this.isValidCheck = true;
+            this.richText = event.target.value;
+        }
+    }
+
+    //Handle initiation of Search and Replace
+    handleOpenSearch(event) {
+        this.selectTrue = !this.selectTrue;
+        if (this.searchButton) this.searchButton = false;
+        else this.searchButton = true;
+    }
+
+    //Search and Replace Search for Value
+    handleSearchChange(event) {
+        this.searchTerm = (event.target.value).replace(/[.*+?^${}()|[\]\\]/g, "\\$&");
+    }
+
+    //Search and Replace Replace with Value
+    handleReplaceChange(event) {
+        this.replaceValue = (event.target.value).replace(/[.*+?^${}()|[\]\\]/g, "\\$&");
+    }
+
+    //Execute Search and REplace
+    searchReplace() {
+        this.oldRichText = this.richText;
+        this.allowRevert = true;
```

Comment:
the convention here is to refer to the block as dirty or not dirty. I'd change 'allowRevert' to 'dirty'

Reply:
Great, thanks.  This will be in next update.